src/main/resources/l10n/JabRef_en.properties

```diff
@@ -476,6 +476,27 @@ Journal\ abbreviations=Journal abbreviations
 Journal\ lists\:=Journal lists:
 Remove\ journal\ '%0'=Remove journal '%0'
 
+Journal\ Information=Journal Information
+Year=Year
+ISSN\ required\ for\ fetching\ journal\ information=ISSN required for fetching journal information
+Allow\ sending\ ISSN\ to\ a\ JabRef\ online\ service\ (SCimago)\ for\ fetching\ journal\ information=Allow sending ISSN to a JabRef online service (SCimago) for fetching journal information
+Please\ enable\ journal\ information\ fetching\ in\ Preferences\ >\ Web\ search=Please enable journal information fetching in Preferences > Web search
+Categories=Categories
+Country=Country
+ISSN=ISSN
+Publisher=Publisher
+h-index=h-index
+SCImago\ ID=SCImago ID
+Error\ while\ fetching\ journal\ information\:\ %0=Error while fetching journal information: %0
+ISSN\ not\ found\ in\ catalog=ISSN not found in catalog
+Citable\ Docs\ (Previous\ 3\ Years)=Citable Docs (Previous 3 Years)
+Cites\ Incoming\ By\ Recently\ Published=Cites Incoming By Recently Published
+Cites\ Outgoing=Cites Outgoing
+Cites\ Outgoing\ Per\ Document=Cites Outgoing Per Document
```

Comment:
Can you do title casing according to Chicago Manual of Style (or APA) here?

    Citations Outgoing per Document